apps/frontend/src/pages/access-management/libs/components/groups-table/groups-table.tsx

```diff
@@ -19,14 +20,21 @@ type Properties = {
 const GroupsTable = ({
 	groups,
 	onDelete,
+	onEdit,
 	onPageChange,
 	onPageSizeChange,
 	page,
 	pageSize,
 	paginationBackground = "primary",
 	totalItemsCount,
 }: Properties): JSX.Element => {
-	const groupColumns = getGroupColumns(onDelete);
+	const groupColumns = getGroupColumns({
+		onDelete,
+		onEdit: (groupId: number) => {
+			const group = groups.find(({ id }) => id === groupId);
+			onEdit(group as GroupGetAllItemResponseDto);
+		},
```

Comment:
let's move it to a separate variable
```suggestion
		onEdit: handleEdit
```